### DISCUSSION TITLE
Documents Copy Email - PDF's not inclueded

### DISCUSSION THREAD

**Discussion by Elteide** [docusealco/docuseal#305] [General]
Hi, 

Since I updated to the last version, the documents are not included in the mail sent to all signers. 
The option is also missing in the personalization settings. There is only "Send emails automatically on completion" but the signed PDFs are not attached.

The link {{submission.link}} is not working for signers, error msg.: You need to sign in or sign up before continuing. 
Require authentication for file download links in the Settings is turned off. 

I hope you can help me.
THX, 
-Matthias

**Comment by omohokcoj** [comment]
@Elteide thanks for reporting this. what is the file size of your signed PDF documents? - email attachments have a limit of 10 MB - if PDF is > 10MB then it won't be included in the attachment.

{{submission.link}} should work if signers specify their email and receive a new link expired link with timestamp to download the files.

**Reply by Elteide** [reply]
Ah okay, thanks for letting me know the limit of 10 MB. I updated yesterday the Form to also include two pictures, may this exceeds the file limits. I will verify this via another Form. 

I have multiple signers all added with names and email addresses, but it doesn't work, error msg.: You need to sign in or sign up before continuing.

**Reply by omohokcoj** [reply]
@Elteide use {{documents.link}} instead of {{submission.link}} 
{{documents.link}} is the link that should be used in the email to the signers  {{submission.link}}  is the link that should be used in emails to admins

**Comment by Elteide** [comment]
Thank you very much, I ran into the 10 MB limit. 
Everything works as I changed the link from {{submission.link}} to {{documents.link}} 😎
